src/applications/common/schemaform/save-in-progress/reducers.js

```diff
@@ -97,6 +98,22 @@ export const saveInProgressReducers = {
       loadedStatus: LOAD_STATUSES.pending
     });
   },
+  SET_PRESTART_STATUS: (state, action) => {
+    const newState = _.set('prestartStatus', action.status, state);
+
+    if (action.status) {
+      newState.prestartExpirationDate = action.expirationDate;
+    }
+
+    if (action.errorMessage) {
```

Comment:
Just personal preference, but I thing having a separate error action is more clear. In the redux dev tools the story is easier to read from the action types e.g. `PENDING` => `SUCCESS` or `PENDING` => `FAILURE`.

Reply:
Thanks @rianfowler, great point, I will create a separate error action.